### ISSUE TITLE
Disappearing PMID's

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#2739]
I noticed a couple of blank lines in the list of papers I have identified as "curatable" but which had not been assigned to a curator. 

<img width="884" alt="Screenshot 2023-06-22 at 08 32 57" src="https://github.com/pombase/canto/assets/7359272/f4325749-9ca9-4e71-acf3-eec05128e1dc">

I re-triaged these as"loaded in error" because I could not locate them in PubMed. But these shouldn't really just 'disappear' if they once existed, and now I am curious as to what they were.

Can we dig the old titles out? I wonder if these are  some of the publications that Pubmed indexed twice in error, BUT in that case, they should still be searchable and lead to the appropriate publication (Also I don't remember this problem beginning this long ago (this is from 2014)

Now I spotted another 2:
<img width="921" alt="Screenshot 2023-06-22 at 08 41 47" src="https://github.com/pombase/canto/assets/7359272/f03d0f83-24e8-4a18-8817-737907553622">

@kimrutherford  is it possible to  pull out the titles from older data? Do we refresh titles (I assumed once things were loaded they would stay the same?)

I will check whether we have any instances of this for curated papers.....

**Comment by ValWood** [comment]
Weird. I wondered whether I could find them in EPMC but they have disappeared without trace....

**Comment by kimrutherford** [comment]
> is it possible to pull out the titles from older data? Do we refresh titles (I assumed once things were loaded they would stay the same?)

Yep, once Canto has the title it doesn't get removed or replaced.  So those PMIDs didn't have titles when loaded.

I checked the dates they were added to Canto.  None of them are recent:

```
PMID:11032848  2012-02-24 05:52:00
PMID:22275432  2012-02-24 05:46:53
PMID:22976396  2012-11-19 00:16:04
PMID:23018174  2012-11-19 00:16:03
PMID:24009652  2013-10-09 03:17:11
PMID:24888876  2014-06-09 10:36:17
PMID:24888877  2014-06-09 10:36:17
PMID:25078613  2014-08-02 00:15:27
PMID:25217569  2014-09-15 00:15:25
PMID:25519574  2014-12-20 01:16:01
PMID:25776683  2015-03-19 01:15:36
PMID:27002032  2016-03-29 00:20:35
PMID:27002055  2016-03-29 00:20:35
PMID:27392239  2016-07-10 00:15:12
PMID:27687092  2016-10-04 00:20:18
PMID:27913728  2016-12-05 01:15:12
PMID:28872942  2017-09-08 00:15:15
```

**Comment by ValWood** [comment]
That's weird though, if they didn't have a title how would I have triaged them as curatable?
I wouldn't have known what they were. Also, I look at that list all the time to assign sessions out or pick sessions to curate. I would definitely have spotted the blank lines before...I'm not sure of many things, but I'm 99.9% sure about this... The _must_ of had titles at some point is it possible the titles are exact duplicates (I'm not sure why that would make a difference,)

**Comment by kimrutherford** [comment]
OK.  I'll have a look in the Canto backups from those dates to see what the titles are.

**Comment by kimrutherford** [comment]
You were right.  Here are the titles from a backup from 2017.  I check a random selection of the titles and found them in PubMed with new PMIDs.

```
PMID:11032848  Nic1p, a relative of bacterial transition metal permeases in schizosaccharomyces pombe, provides nickel ion for urease biosynthesis
PMID:22275432  A Meiotic Actin Ring (MeiAR) Essential for Proper Sporulation in Fission Yeast.
PMID:22976396  Corrigendum: The smooth and stable operation of centromeres [Genes Genet. Syst. (2012) 87, p. 63-73].
PMID:23018174  Multiple phosphorylation of Rad9 by CDK is required for DNA damage checkpoint activation.
PMID:24009652  Correction: Polypeptone Induces Dramatic Cell Lysis in ura4 Deletion Mutants of Fission Yeast.
PMID:24888876  Fission Yeast RecQ Helicase Rqh1 Is Required for the Maintenance of Circular Chromosomes.
PMID:24888877  Fission Yeast Pot1 and RecQ Helicase Are Required for Efficient Chromosome Segregation.
PMID:25078613  Correction: Genetic and Metabolomic Dissection of the Ergothioneine and Selenoneine Biosynthetic Pathway in the Fission Yeast, S. pombe, and Construction of an Overproduction System.
PMID:25217569  Correction for Kuscu et al., CRL4-like Clr4 complex in Schizosaccharomyces pombe depends on an exposed surface of Dos1 for heterochromatin silencing.
PMID:25519574  The role of nitric oxide in the fission yeast Schizosaccharomyces pombe under oxidative stress conditions.
PMID:25776683  CodingQuarry: highly accurate hidden Markov model gene prediction in fungal genomes using RNA-seq transcripts.
PMID:27002032  The Src family of protein tyrosine kinases: regulation and functions.
PMID:27002055  Investigations into the control of cell form and polarity: the use of morphological mutants in fission yeast.
PMID:27392239  The effect of magnesium on mitotic spindle formation in Schizosaccharomyces pombe.
PMID:27687092  Radiation checkpoints in model systems.
PMID:27913728  Fission yeast Stn1 is crucial for semi-conservative replication at telomeres and subtelomeres.
PMID:28872942  Radiation Checkpoints in Model Systems.
```

**Comment by ValWood** [comment]
[Nucleic Acids Res.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388396/#) 2017 Feb 17; 45(3): 1255–1269.
Published online 2016 Nov 29. doi: [10.1093/nar/gkw1176](https://doi.org/10.1093%2Fnar%2Fgkw1176)
PMCID: PMC5388396
PMID: [28180297](https://pubmed.ncbi.nlm.nih.gov/28180297)
Fission yeast Stn1 is crucial for semi-conservative replication at telomeres and subtelomeres
at least, has another entry.

I wonder if they are all  Corrigendum:  eitherway they shouldn't disappear
Will contact ePMC and PubMed

**Comment by ValWood** [comment]
I moved any which are flagged as "curatable" to "erratum" because there seems to be another version of these.
Once I hear back from PubMed and/or EPMC I can review this.
Transferring to curation tracker.
